NewsFeed/views/CCBRNewsViewController.m

```diff
@@ -79,13 +82,17 @@ - (void)viewDidLoad {
                                                     bundle:nil]
           forCellWithReuseIdentifier:kCCBRNewsSmallCardView];
     
-    [self updateUI];
+    [self updateUIWithError:@""];
 }
 
-- (void)updateUI {
-    self.collectionView.hidden = self.viewModel.collectionViewHidden;
-    self.errorMessageLabel.hidden = self.viewModel.errorMessageLabelHidden;
-    [self.collectionView reloadData];
+- (void)updateUIWithError:(NSString*)error {
+    
+    self.collectionView.hidden = self.viewModel.collectionViewHidden ;
+    if ([error isEmpty]) [self.collectionView reloadData];
+    
+    self.errorMessageLabel.hidden = [error isEmpty];
+    self.errorMessageLabel.text = error;
```

Comment:
Because View/ViewController should be as dumb as possible, these logic should be moved to ViewModel.